modules/backend/lang/zh-cn/lang.php

```diff
@@ -501,7 +553,7 @@
         'label' => '媒体查找器',
         'default_prompt' => '点击 %s 按钮查找媒体项'
     ],
-   'media' => [
+    'media' => [
```

Comment:
@bennothommo any ideas why CI approve space? Strange)

Reply:
@wenlong-date could you remove space before 'media', doesn't seem to be correct with whitespace.

Reply:
@w20k The extra space puts it back in correct alignment, from what I can tell, which is why it passes.

Reply:
@bennothommo you're right, need a rest or two :) Thanks!